src/AppConstants.js

```diff
@@ -63,23 +64,43 @@ export const SEVERITY_MAP = {
     'low-risk': 1
 };
 export const RISK_OF_CHANGE_DESC = {
-    1: intlHelper(intl.formatMessage(messages.riskOfChangeTextOne), intlSettings),
-    2: intlHelper(intl.formatMessage(messages.riskOfChangeTextTwo), intlSettings),
-    3: intlHelper(intl.formatMessage(messages.riskOfChangeTextThree), intlSettings),
-    4: intlHelper(intl.formatMessage(messages.riskOfChangeTextFour), intlSettings)
+    1: intlHelper(intl.formatMessage(
+        messages.riskOfChangeTextOne, { strong(str) { return <strong>{str}</strong>; } }
```

Comment:
_ducking_ cant help but wonder if the `strong(str) { return <strong>{str}</strong>;` could be declared once, used everywhere 🤔  (not blocking merge of this on that or anything... just 🤔 )

Reply:
I like that idea 🤔 would something like that belong as a new file in the `Utilities` directory?

Reply:
OOO yeah! in utilities could work... make it like an intl helper file or something, we could kill ah lotta repetition that way... cuz <strong> s used in quite a few places